Bottom: c6971db5e045ef3fdb59a0ea638b26a22a195072
Top:    dbdb4d0bf058da11dcf27c5f068b02e823cbe6ce
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-01-31 11:44:20 -0600

Update environment file and usage

Updates the docker-compose environment file, improving consistency
and extracting several variables from docker-compose.yml.



---
diff --git a/docker/buildbot-worker-ntel/buildbot/buildbot.tac b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
index 3ba7179..472f609 100644
--- a/docker/buildbot-worker-ntel/buildbot/buildbot.tac
+++ b/docker/buildbot-worker-ntel/buildbot/buildbot.tac
@@ -15,13 +15,13 @@ application = service.Application('buildbot-worker')
 
 application.setComponent(ILogObserver, FileLogObserver(sys.stdout).emit)
 # and worker on the same process!
-buildmaster_host = os.environ.get("BUILDMASTER", 'localhost')
-port = int(os.environ.get("BUILDMASTER_PORT", 9989))
-workername = os.environ.get("WORKERNAME", 'docker')
-passwd = os.environ.get("WORKERPASS")
+buildmaster_host = os.environ.get("BUILDBOT_MASTER", 'localhost')
+port = int(os.environ.get("BUILDBOT_WORKER_PORT", 9989))
+workername = os.environ.get("BUILDBOT_WORKER_NTEL", 'docker')
+passwd = os.environ.get("BUILDBOT_WORKER_PASSWORD")
 
 # delete the password from the environ so that it is not leaked in the log
-blacklist = os.environ.get("WORKER_ENVIRONMENT_BLACKLIST", "WORKERPASS").split()
+blacklist = os.environ.get("BUILDBOT_ENVIRONMENT_BLACKLIST", "BUILDBOT_WORKER_PASSWORD").split(',')
 for name in list(os.environ.keys()):
     for toremove in blacklist:
         if fnmatch.fnmatch(name, toremove):
